node_test/rpc_test.go

```diff
@@ -96,10 +79,9 @@ func executeNRpcTest(t *testing.T, connectionType transport.TransportType, n int
 		manVoucherStr = ""
 	}
 	logFile := fmt.Sprintf("test_%d_rpc_clients_over_%s%s.log", n, connectionType, manVoucherStr)
-	logDestination := logging.NewLogWriter("../artifacts", logFile)
-	defer logDestination.Close()
-	logger := testLogger(logDestination)
-	logger.Info().Msgf("Starting test with %d clients", n)
+	logging.SetupDefaultFileLogger(logFile, slog.LevelDebug)
+
+	slog.Info("Starting test", "num-clients", n)
```

Comment:
Since we've configured a `DefaultLogger` that logs out to a file, these statements will be written to the file.